DashAI/back/tasks/tabular_classification_task.py

```diff
@@ -12,11 +12,18 @@ class TabularClassificationTask(BaseTask):
     name: str = "TabularClassificationTask"
     schema: dict = {
         "inputs_types": [ClassLabel, Value],
-        "outputs_types": [ClassLabel, Value],
+        "outputs_types": [ClassLabel],
         "inputs_cardinality": "n",
         "outputs_cardinality": 1,
     }
 
     @staticmethod
     def create():
         return TabularClassificationTask()
+
+    def prepare_for_task(self, datasetdict: DatasetDict):
```

Comment:
If the method extends the abstract of the super class, it still has to be documented according to its specific functionality.